### PR TITLE
Fix to #2236 - tensor.numpy() checks that no positional arguments are passed.

### PR DESCRIPTION
**Pull request by ghostFaceKillah**
Created a check in tensor.numpy() that checks if no positional arguments are passed.

Fixes:
https://github.com/pytorch/pytorch/issues/2236

### CODE REVIEW

File: torch/csrc/generic/methods/TensorSerialization.cwrap [hunk 1]
```diff
@@ -70,6 +70,11 @@ PyObject * THPTensor_(toNumpy)(THPTensor *self, PyObject *args) {
   std::unique_ptr<npy_intp[]> sizes;
   std::unique_ptr<npy_intp[]> strides;
 
+  if (!PyArg_ParseTuple(args, "")) {
```
Comment (apaszke) [inline torch/csrc/generic/methods/TensorSerialization.cwrap]: A faster way to do it would be `THPUtils_assert(PyTuple_SIZE(args) == 0, "numpy() ...");`

**Review by apaszke — COMMENTED** [review]
Thanks for the PR! I've added a suggestion that would make the check faster

**Review by apaszke — APPROVED** [review]

**Comment by ghostFaceKillah** [comment]
Thanks for the suggestion!

**Pull request merged**